api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java

```diff
@@ -278,7 +278,7 @@ public Obs voidObs(Obs obs, String reason) throws APIException {
 	 * @throws APIException
 	 */
 	public Obs unvoidObs(Obs obs) throws APIException {
-		return dao.saveObs(obs);
+		return Context.getObsService().saveObs(obs,"unvoid obs");
```

Comment:
I see, I guess it would be nice to discuss this on a design call about what the proper behavior should be, I would Imagine that if an Obs is voided and later unvoided with changes as it was getting voided or unvoided that would technically constitute a change but it would fairly be complex to track the change.